localisation/00_wc_sad.csv

```diff
@@ -119,12 +119,12 @@ cenarion_circle_rank_2_male;Hierophant;;Hierophant;;;;;;;;;;;x
 cenarion_circle_rank_2_female;Hierophant;;Hierophantin;;;;;;;;;;;x
 cenarion_circle_rank_3_male;Instructor;;Lehrmeister;;;;;;;;;;;x
 cenarion_circle_rank_3_female;Instructor;;Lehrmeisterin;;;;;;;;;;;x
-cenarion_circle_rank_4_male;Archdruid;;Erzdruide;;;;;;;;;;;x
-cenarion_circle_rank_4_female;Archdruid;;Erzdruidin;;;;;;;;;;;x
-cenarion_circle_rank_5_male;Leader;;Anf�hrer;;;;;;;;;;;x
-cenarion_circle_rank_5_female;Leader;;Anf�hrerin;;;;;;;;;;;x
+cenarion_circle_rank_4_male;Keeper;;H�ter;;;;;;;;;;;x
+cenarion_circle_rank_4_female;Keeper;;H�terin;;;;;;;;;;;x
+cenarion_circle_rank_5_male;Archdruid;;Erzdruide;;;;;;;;;;;x
+cenarion_circle_rank_5_female;Archdruid;;Erzdruidin;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
-sisterhood_of_elune;Sisterhood of Elune;;Schwesternschaft von Elune;;;;;;;;;;;x
+sisterhood_of_elune;the Sisterhood of Elune;;Schwesternschaft von Elune;;;;;;;;;;;x
```

Comment:
the the

<details>

![image](https://user-images.githubusercontent.com/46576860/64913663-f20dfc80-d74c-11e9-8419-0597f6b8074d.png)

</details>

Reply:
Fixed!